source/sass/blocks/catalog-add.scss

```
@@ -86,7 +86,7 @@
 }
 
 .catalog-add__promo {
-  background-color: #71b548;
+  background-image: url("/img/gift-mobile.jpg");
```

```suggestion
  background-image: url("../img/gift-mobile.jpg");
```
Нужно исправить `url` для всех фоновых изображений, где нет `..` перед `/`.